float32 surge
float32 surge_speed
float32 surge_time
int8 is_surge

float32 sway
float32 sway_speed
float32 sway_time
int8 is_sway

float32 yaw
float32 yaw_speed
float32 yaw_time
int8 is_yaw

float32 depth
float32 depth_speed
float32 depth_time
int8 is_depth

int8 isAbsolute
int8 wait_for_max_timer_to_timeout
int8 mode_after_last_cmd
